src/modules/commentsApi.js

```diff
@@ -0,0 +1,58 @@
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const appId = '8ge52RZ4xe8gKMYNyxDN';
+
+// Function to create a new app
+/* async function createApp() {
+  const url = `${baseUrl}apps/`;
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+    });
+    if (response.ok) {
+      const data = await response.text(); // Get the response as text
+      appId = data.trim(); // Trim whitespace from the response
+      console.log('App created. App ID:', appId);
+    } else {
+      console.error('Failed to create app:', response.status, response.statusText);
+    }
+  } catch (error) {
+    console.error('Error creating app:', error);
+  }
+} */
```

Comment:
- [ ] Fantastic work on implementing the project requirements! 👏 Your codebase looks clean and well-organized, but I have a small suggestion to further enhance its readability and maintainability.

   - I noticed that there are some commented-out code sections in your codebase. While it's understandable that you might want to keep them for future reference or as a backup, leaving them in the code can make it challenging for others (and even yourself) to read and understand the codebase thoroughly.

   - As a best practice, I recommend removing all commented-out code from your project. By doing so, you'll streamline the codebase and make it much easier for everyone involved to grasp the project's logic and flow.

Keep up the excellent work. Happy coding! 😊🚀